src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -503,24 +503,836 @@ A table with a full reference of error codes, messages and description:
+                    "attributeCategoryOptions": "xYerKDKCefk",
+                    "dataValues": [],
+                    "notes": []
+                }
+            ],
+            "relationships": [],
+            "attributes": [],
+            "notes": []
+        }
+    ],
+    "programOwners": [
+        {
+            "orgUnit": "g8upMTyEZGZ",
+            "trackedEntity": "IzHblRD2sDH",
+            "program": "ur1Edk5Oe2n"
+        }
+    ]
+}
+```
+
+### Events (`GET /api/tracker/events`)
+
+Two endpoints are dedicated to events:
+
+- `GET /api/tracker/events`
+    - retrieves events matching given criteria
+- `GET /api/tracker/events/{id}`
+    - retrieves an event given the provided id
+
+#### Events Collection endpoint `GET /api/tracker/trackedEntities`
+
+Purpose of this endpoint is to retrieve events matching client-provided criteria.
+
+It's intended to return a collection of events with pagination TODO [VERIFY USE CASES].
+
+|request parameter|type|allowed values|description|
+|---|---|---|---|
+|`program`|`String`|`uid`| Identifier of program|
+|`programStage`|`String`|`uid`| Identifier of program stage|
+|`programStatus`|`enum`| `ACTIVE`&#124;`COMPLETED`&#124;`CANCELLED`| Status of event in program | 
+|`followUp`|`boolean`| `true`&#124;`false` | Whether event is considered for follow up in program. Defaults to `true`|
+|`trackedEntityInstance`|`String`|`uid`| Identifier of tracked entity instance|
+|`orgUnit`|`String`|`uid`| Identifier of organisation unit|
+|`ouMode` see [ouModes](#Request-parameters-for-Organisational-Unit-selection-mode)|`String`| `SELECTED`&#124;`CHILDREN`&#124;`DESCENDANTS`|	Org unit selection mode| 
+|`occurredAfter`|`DateTime`| |	Only events newer than this date|
+|`occurredBefore`|`DateTime`| | Only events older than this date|
+|`status`|`String`|`COMPLETED`&#124;`VISITED`&#124;`SCHEDULED`&#124;`OVERDUE`&#124;`SKIPPED` | Status of event|
+|`occurredAfter`|`DateTime`| | Filter for events which were occurred after this date.|
+|`occurredBefore`|`DateTime`| | Filter for events which were occurred up until this date.|
+|`scheduledAfter`|`DateTime`| | Filter for events which were scheduled after this date.|
+|`scheduledBefore`|`DateTime`| | Filter for events which were scheduled up until this date.|
+|`updatedAfter`|`DateTime`| | Filter for events which were updated after this date. Cannot be used together with `updatedWithin`.|
+|`updatedBefore`|`DateTime`| | Filter for events which were updated up until this date. Cannot be used together with `updatedWithin`.|
+|`updatedWithin`|`Duration`| | Include only items which are updated within the given duration.<br><br> The format is [ISO-8601#Duration](https://en.wikipedia.org/wiki/ISO_8601#Durations)|
+|`skipMeta`|`Boolean`| `true`&#124;`false` | Exclude the meta data part of response (improves performance)|
+| TODO (?) `dataElementIdScheme`|`String`| `UID`&#124;`CODE`&#124;`ATTRIBUTE:{ID}`|	Data element ID scheme to use for export.|
```

Comment:
IdSchemes:

IdScheme determine what identifier we should work with. We have a couple of standard ones: AUTO(UID), UID, UUID(UID), CODE, NAME. These are all unique.


IdSchemes are currently only affecting metadata, so instead of Program: "<uid>" it would be returned as Program: "<code>" for example.

We can talk more about IdSchemes later if you want.

Also: Attributes. Attributes are the more complex IdSchemes. They are not the same as "TrackedEntityAttribute", but a way to extend the metadata model. When metadata is extended with an attribute, that attribute can be used as an IdScheme. Then you also need to specify the uid of the attribute to use.

Reply:
This comment looks weird.

In any case, I can add the required info for these params myself :)